src/dashboard/trulens/dashboard/ux/components.py

```
@@ -79,8 +80,21 @@ def draw_metadata_and_tags(
 ) -> str:
     if isinstance(metadata, Dict):
         metadata["tags"] = tags
+
+        # Make the boolean column names more user-friendly
+        if metadata.get(dashboard_constants.EXTERNAL_APP_COL_NAME) is not None:
+            metadata["External App?"] = metadata.pop(
+                dashboard_constants.EXTERNAL_APP_COL_NAME
+            )
+
+        if metadata.get(dashboard_constants.PINNED_COL_NAME) is not None:
+            metadata["Pinned?"] = metadata.pop(
+                dashboard_constants.PINNED_COL_NAME
+            )
+
         return dict_to_md(metadata)
     else:
+        # We shouldn't ever get here since types_schema.Metadata is a Dict.
```

can we just throw an error then?

---

okay - maybe let me properly deprecate the experimental path that leads here in a separate PR.